### PR TITLE
Move testing utilities to the platform directory

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets-contrib/issues/53

This moves testing utilities into the platform directory. All the changes in here are comments and path changes, so there's not much to review. And as long as the build passes, everything is set up correctly.

### CODE REVIEW

File: src/platform/testing/e2e/nightwatch-commands/printLogs.js [hunk 1]
```diff
@@ -13,8 +13,8 @@
  *                           some logs to be skipped. For all log levels, See
  *                           https://github.com/SeleniumHQ/selenium/wiki/Logging#log-levels
  *                           If no level is provided, all levels are output.
- * @param {bool}   printAll  Prints all the logs from 0 on. In case some logs
- *                           got skipped because of log level specificity.
+ * @param {bool}   [printAll=false]  Prints all the logs from 0 on. In case some logs
+         *                           got skipped because of log level specificity.
```
Comment (U-DON) [inline src/platform/testing/e2e/nightwatch-commands/printLogs.js]: I think the preceding spaces are accidental?

File: src/platform/testing/e2e/nightwatch-commands/log.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  *  logging as the browser is running through the test due to the way nightwatch
  *  runs all the code beforehand.
  *
- * @param {String} message The message you want to log to the console
+ * @param {String} [message=''] The message you want to log to the console
```
Comment (U-DON) [inline src/platform/testing/e2e/nightwatch-commands/log.js]: Capitalized `String` or lowercase `string`? Looks like you were going for the latter for consistency.

File: src/platform/testing/unit/schemaform-utils.jsx [hunk 3]
```diff
@@ -135,7 +164,14 @@ function printTree(node, level = 0, isLastChild = true, padding = '') {
...
 }
 
+/**
+ * Gets the DOM node associated with the form tree passed in.
+ *
+ * @param {React.Element} form The root level of a rendered form
+ * @returns {object} An DOM node for the form, with added helper methods
+ */
 export function getFormDOM(form) {
+  // eslint-disable-next-line react/no-find-dom-node
```
Comment (U-DON) [inline src/platform/testing/unit/schemaform-utils.jsx]: Didn't you override this already in `.eslintrc`?

File: test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx [hunk 4]
```diff
@@ -3,12 +3,12 @@ import { expect } from 'chai';
...
 import { mount } from 'enzyme';
 
-import { DefinitionTester, fillData } from '../../../util/schemaform-utils.jsx';
+import { DefinitionTester, fillData } from '../../../../src/platform/testing/unit/schemaform-utils.jsx';
 import formConfig from '../../../../src/js/disability-benefits/526EZ/config/form.js';
 import initialData from '../schema/initialData.js';
 
-describe('Disability benefits 526EZ VA facility', () => {
-  const { schema, uiSchema, arrayPath } = formConfig.chapters.supportingEvidence.pages.privateMedicalRecordsRelease;
+xdescribe('Disability benefits 526EZ VA facility', () => {
```
Comment (U-DON) [inline test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx]: Intentionally disabled?

**Review by U-DON — APPROVED** [review]
I like the reorganization of the helpers into unit and E2E categories. LGTM and just had some minor comments.

File: test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx (hunk 4 above)
Comment (jbalboni) [reply test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx]: Yes, this file was missing the spec extension and not being run. I fixed that and the test is failing, so I explicitly ignored them. @cvalarida may want to take a look at this.

File: src/platform/testing/unit/schemaform-utils.jsx (hunk 3 above)
Comment (jbalboni) [reply src/platform/testing/unit/schemaform-utils.jsx]: Yep, missed removing this afterward.

**Comment by jbalboni** [comment]
I don't think I'm going to do it right now, but it probably makes sense to move mockapi.js and test-server.js into /config, or some other place specific to build config and tooling.

**Review by ncksllvn — APPROVED** [review]

File: test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx (hunk 4 above)
Comment (cvalarida) [reply test/disability-benefits/526EZ/config/privateMedicalRecordsRelease.unit.spec.jsx]: I think @cehsu said she wrote this and was looking into it.
Thanks for keeping us in the loop! :raised_hands:

**Pull request merged**